util/minetester/clean.sh

```diff
@@ -0,0 +1,2 @@
+rm -r build
+rm bin/*
```

Comment:
I don't know what the best practice is for cleaning up, but I would probably not delete the binaries

Reply:
see above